contracts/context-config/src/mutate.rs

```diff
@@ -53,9 +55,39 @@ impl ContextConfigs {
                 ContextRequestKind::Revoke { capabilities } => {
                     self.revoke(&request.signer_id, context_id, capabilities.into_owned());
                 }
+                ContextRequestKind::DeployProxyContract {} => {
+                    self.deploy_proxy_contract(&request.signer_id, context_id);
```

Comment:
we can add an `update` method, but the initial deployment should happen on context creation